### ISSUE TITLE
Oops on execute "systemctl restart nginx"

### ISSUE THREAD

**Issue by ancienter** [gdraheim/docker-systemctl-replacement#108]
ERROR:systemctl: !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
ERROR:systemctl: Oops, 1 unsupported directory settings. You need to create those before using the service.
ERROR:systemctl: !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by DenisLuka21** [comment]
use **systemctl start nginx -vvv**. You see broken param in **nginx.service** file. Delete **PrivateTmp=true** from **nginx.service** file and after use **killall nginx**. Now you can start nginx

**Comment by ancienter** [comment]
it's down,it works now

**Comment by ancienter** [comment]
thinks

**Comment by lhenry7** [comment]
Hi, new to this project. I have a question about the default PrivateTemp=true setting(also set for apache). Is there some other setup required to make the default work, or is the only option to change the service file after httpd install and before systemctl start?

**Comment by gdraheim** [comment]
On the develop branch, I have just changed the implementation - after looking into the details of what PrivateTmp should do, it seems that we can just ignore that setting in a docker container.

I may find the time to release it to master over the holidays.

**Comment by lhenry7** [comment]
@gdraheim - Sounds great, I look forward to giving it a try once ready.

**Comment by ancienter** [comment]
i'am sorry,i have a wrong config，i found it then it works well now。

**Comment by ancienter** [comment]
Yes, it is caused by this problem. I found the problem after consulting the document, so it can work now. Thank you very much for your answer

**Comment by ancienter** [comment]
Yes, that's the problem. I checked the documentation and found the problem. I configured it manually, so it works now
